src/Apps/Collect2/Routes/Collect/index.tsx

```diff
@@ -1,21 +1,25 @@
-import { Box, Flex, Sans, Serif } from "@artsy/palette"
+import { Separator, Serif } from "@artsy/palette"
+import { Collect_marketingCollections } from "__generated__/Collect_marketingCollections.graphql"
 import { Collect_viewer } from "__generated__/Collect_viewer.graphql"
 import { CollectFilterFragmentContainer as ArtworkGrid } from "Apps/Collect2/Components/Base/CollectFilterContainer"
 import { SeoProductsForArtworks } from "Apps/Collect2/Components/Seo/SeoProductsForArtworks"
 import { AppContainer } from "Apps/Components/AppContainer"
+import { withSystemContext } from "Artsy"
 import { track } from "Artsy/Analytics"
 import * as Schema from "Artsy/Analytics/Schema"
 import { FrameWithRecentlyViewed } from "Components/FrameWithRecentlyViewed"
+import { CollectionsHubsNavFragmentContainer as CollectionsHubsNav } from "Components/v2/CollectionsHubsNav"
```

Comment:
Is it appropriate to place this component inside of `Apps/Collect2/Collection/Components`? Like, ATM, will this *only* exist on collections-related pages? If so, thoughts on keeping it close until it becomes more general? Similar to the other CollectionsHub rails that were relocated. 

Also, just had another thought re the links -- if this component _isn't_ only on collect-related pages, then there won't be a global routing context, and the `<Link>` component won't work. 

(Which just gave me an idea to test for `router`, and as a failsafe fall back to a regular `a` tag 🤔).

Reply:
These entry points will also live on the homepage.

Reply:
Ok cool, then we'll have to implement this conditional Link thing. 

@pepopowitz, if you wouldn't mind rolling with the link changes mentioned below (and haven't already added them to the artsy.net homepage) I'll follow up with a PR that makes the link component a bit more safe.

Reply:
Yes, I think this would be a logical move. It started to get in my way in `v2/components` as I started working on the next story, which is to put something similar but different on the home page.

Reply:
Oh, my browser had a stale conversation, from before @xtina-starr 's comment. As of right now, the entry-points on home look pretty different than these, but there were some changed designs that I haven't looked at very closely. For now I'll leave it where it is, and when we get to the home page we can decide if this is a common component or unique to the collect page.